test/object-store/realm.cpp

```diff
@@ -565,7 +565,7 @@ TEST_CASE("Get Realm using Async Open", "[asyncOpen]") {
                 return bool(realm_ref);
             });
             SharedRealm realm = Realm::get_shared_realm(std::move(realm_ref));
-            realm->write_copy_without_client_file_id(config3.path, BinaryData());
+            realm->write_copy(config3.path, BinaryData());
```

Comment:
Is any of the checks down below also checking if the file id was not copied over?

Reply:
I have added at test that checks that we have obtained a new client file id which is an indication that there was no id stored in the file.